xdg.go

```diff
@@ -113,6 +120,16 @@ func DataFile(relPath string) (string, error) {
 	return baseDirs.dataFile(relPath)
 }
 
+// StateFile returns a suitable location for the specified data (mutable) file.
```

Comment:
I think we should use the term `volatile` instead of `mutable`.

Reply:
Fixed